.github/workflows/ci.yml

```diff
@@ -21,5 +21,5 @@ jobs:
     - run: npm run check
     - env:
         VALUESET_KEY: ${{ secrets.VALUESET_KEY }}
-      run: unzip -P $VALUESET_KEY test/fixtures/EXM130-8.0.000/EXM130-8.0.000-bundle.zip
-    - run: rm EXM130-8.0.000-bundle.json
+    - run: ./setup.sh
```

Comment:
The EXM130 package still needs to be extracted for the unit tests to run AFAIK.

Reply:
These two script references are likely incorrect as they are not in the current folder.